### DISCUSSION TITLE
Can different node pools be in different resource groups?

### DISCUSSION THREAD

**Discussion by charalamm** [Azure/AKS#3755] [Q&A]
I would like to associate costs of each pod in a namespace with the project that it is running for so I was thinking to spawn pods of each project in a separate node pool and have each node pool in a project-dedicated resource group. However I do not find a way to add a node pool to a different resource group than the resource group of aks. Is it possible?

**Comment by PixelRobots** [comment]
It might be wise to look at KubeCost instead. https://www.kubecost.com/

**Comment by charalamm** [comment]
We used [opencost](https://www.opencost.io/) at the end which if I understand correctly it is heavily based on kubecost
